python-sdk/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+markers =
+    rust_only: mark a test as only passing on Python-on-Rust SDK.
```

Comment:
☝️ All tests without this marker pass on both python-on-rust and native python implementations